### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by CalmAfterReboot** [xtekky/gpt4free#362] [Ideas]
I have try the tool and i think is doing the job well however i come across autoGPT which is Open-Source also , how could we generate an API key from Gpt4free so we can just plug it in and try it out ?

**Comment by Seredeep** [comment]
There is a WebApp for trying it out. If you want to code on top of it you would have to set up the project locally, build your custom API around it and then interact with it

**Comment by comejv** [comment]
There's another website that uses GPT4 you could add : [Phind](https://phind.com). Has internet access.

When you switch "expert" mode on it's GPT4, idk what's used otherwise

**Reply by mitchcapper** [reply]
phind was previously part of this repo and removed at their request #153
